include/powsybl/iidm/converter/ExportOptions.hpp

```diff
@@ -110,6 +110,15 @@ class ExportOptions {
      */
     const TopologyLevel& getTopologyLevel() const;
 
+    /**
+     * Return true if one (or more) extension of the given list should be exported
+     *
+     * @param extensions the list of extensions to be checked.
+     *
+     * @return true if at least one extension should be exported, false otherwise
+     */
+    bool hasAtLeastOneExtension(const std::set<std::string>& extensions) const;
```

Comment:
Looking to the java code, there is an `AbstractOptions` class to share code between `ImportOptions` and `ExportOptions`. Even it's not used, I propose to refactor the option classes to make the code more similar to the java.

Reply:
Postponed. I created the issue #140 to report the fix.